<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>0</y>
     <width>641</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Cyber Test Log Detail</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="windowModality" >
    <enum>Qt::NonModal</enum>
   </property>
   <property name="geometry" >
    <rect>
     <x>910</x>
     <y>460</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Close</set>
   </property>
   <property name="centerButtons" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Detail Data</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="detailTable" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>351</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>80</y>
     <width>591</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>600</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel" >
     <x>608</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
